### PR TITLE
Adding Parser section to README.md

### PR DESCRIPTION
**Pull request by JoZie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,22 @@ make install
...
 ```
 
+The Parser
+----------
+ngp implements it's own native source code parser.
+However you can use an external tool for parsing your source code.
+Currently `ag` and `git grep` are supported.
+
+The default parser can be specified with the .ngprc file.
+It defaults to "nat", the native parser.
```
Comment (jonathanklee) [inline README.md]: I don't get this sentence.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,22 @@ make install
...
+Changing the 'default_parser' settings to "ag" or "git" sets the specified tool as parser.
```
Comment (jonathanklee) [inline README.md]: s/settings/setting

**Comment by jonathanklee** [comment]
Nice 👍

File: README.md (hunk 1 above)
Comment (JoZie) [reply README.md]: I totally agree ;)

**Comment by JoZie** [comment]
@jonathanklee The more the text is laying around the better it will get.

**Pull request merged**